mobile/cafe.go

```diff
@@ -18,48 +19,60 @@ func (m *Mobile) RegisterCafe(host string) error {
 }
 
 // CafeSessions calls core CafeSessions
-func (m *Mobile) CafeSessions() (string, error) {
+func (m *Mobile) CafeSessions() ([]byte, error) {
 	if !m.node.Started() {
-		return "", core.ErrStopped
+		return nil, core.ErrStopped
```

Comment:
Tried this out, not being a go or c expert with pointers, but it seems to work. The fact that methods that return `error` can return `nil` when there is no error made me want to try returning `nil` instead of `make([]byte, 0)` in cases where there is no actual byte array to return.

Reply:
Cool, makes sense to me. The "zero" value for a slice is `nil`